### PR TITLE
CLOUDP-167797:Migrate Online Archive CLI stores to v2 sdk

### PR DESCRIPTION
**Pull request by swattyT**
## Proposed changes

_Jira ticket:_ CLOUDP-# [CLOUDP-167797](https://jira.mongodb.org/browse/CLOUDP-167797)

Closes #[issue number]

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by swattyT — COMMENTED** [review]
onlineArchives start and pause commands throwing error:
https://jira.mongodb.org/browse/CLOUDP-168574

**Comment by wtrocki** [comment]
We should use get and post - it should ublock that PR.
Fix landed in the upstream (requires dev sdk)

File: internal/cli/atlas/clusters/onlinearchive/start_test.go [hunk 1]
```diff
@@ -48,10 +49,10 @@ func TestStart_Run(t *testing.T) {
 		store: mockStore,
 	}
 
-	paused := false
-	expected := &mongodbatlas.OnlineArchive{
-		ID:     updateOpts.id,
-		Paused: &paused,
+	// paused := false
```
Comment (wtrocki) [inline internal/cli/atlas/clusters/onlinearchive/start_test.go:52]: Same as above

File: internal/cli/atlas/clusters/onlinearchive/start.go [hunk 2]
```diff
@@ -44,13 +45,12 @@ func (opts *StartOpts) initStore(ctx context.Context) func() error {
...
+var startTemplate = "Online archive '{{.Id}}' started.\n"
 
 func (opts *StartOpts) Run() error {
-	paused := false
-	archive := &atlas.OnlineArchive{
-		ID:     opts.id,
-		Paused: &paused,
+	archive := &atlasv2.OnlineArchive{
+		Id:    &opts.id,
+		State: pointer.Get("ACTIVE"),
```
Comment (swattyT) [inline internal/cli/atlas/clusters/onlinearchive/start.go:53]: Locally tested the Pause and Start commands and it worked fine.

**Review by wtrocki — APPROVED** [review]

**Review by matt-condon — APPROVED** [review]
LGTM!

**Pull request merged**